ipld/amt/Cargo.toml

```diff
@@ -1,11 +1,11 @@
 [package]
-name = "ipld_amt"
+name = "fvm_ipld_amt"
 description = "Sharded IPLD Array implementation."
-version = "1.0.0"
+version = "0.1.0"
 license = "MIT OR Apache-2.0"
-authors = ["ChainSafe Systems <info@chainsafe.io>"]
+authors = ["Filecoin Core Devs"]
```

Comment:
ChainSafe was the original author, so we need to keep then in the author list.

Reply:
In my head, Filecoin Core Devs is a wider group that _includes_ ChainSafe Systems, but happy to make it additive.

Reply:
Let's make it additive. It's good to keep history for stuff like this.